README.md

```diff
@@ -296,6 +296,23 @@ uuidValidate('not a UUID'); // ⇨ false
 uuidValidate('6ec0bd7f-11c0-43da-975e-2a8ad9ebae0b'); // ⇨ true
 ```
 
+Using `validate` and `version` together it is possible to do per-version validation, e.g. validate for only v4 UUIds.
+
+```javascript
+import { version as uuidVersion } from 'uuid';
+import { validate as uuidValidate } from 'uuid';
```

Comment:
I guess runmd stumbles when you put both imports into one line? Because we actually just [hack our way around ESM](https://github.com/broofa/runmd/commit/cdeb284a3dc3c3b48b6626167058a2ea68987f7a)…

Reply:
Hahah yeah it wasn't happy about it at all

Reply:
> Want me to submit it as an issue over on runmd?

Yes, please.   I'm aware of the issue but need to revisit it to see if there's a better solution than the one ctavan and I hacked in a while ago.